kedro/runner/parallel_runner.py

```diff
@@ -274,6 +275,7 @@ def _run(  # pylint: disable=too-many-locals,useless-suppression
         Args:
             pipeline: The ``Pipeline`` to run.
             catalog: The ``DataCatalog`` from which to fetch data.
+            hook_manager: The ``PluginManager`` to activate hooks.
```

Comment:
Also, there's inconsistent backticking of `PluginManager` (as well as other stuff). I copied this from `kedro/runner/runner.py`, but I think it should be single backticks, right (as per https://stackoverflow.com/a/57302493, which I have always used as my reference, for better or for worse).

Reply:
This is a bad source of reference in this case. I think what I wrote here is the correct one: https://github.com/kedro-org/kedro/pull/1130

Reply:
Sorry, my mind just goes back to the bad reference/forgot your comment, even though I apparently 👍ed it. 😅

Reply:
Let me figure out the answer to https://github.com/kedro-org/kedro/pull/1593#discussion_r894484248 before adding more commits on top, just in case going to separate it out.

Reply:
Done!